packages/dev/core/src/Meshes/mesh.vertexData.ts

```diff
@@ -114,6 +114,8 @@ export class VertexData {
      */
     public static readonly DEFAULTSIDE = 0;
 
+    private static _UniqueIDGenerator = 0;

```

Comment:
If it didn't have to be consecutive, you could simply use the scene unique if mechanism.

Reply:
Ignore that, the scene is not really available when it's constructed

Reply:
yep :) less dependencies ;D